resource-definitions/k8s-cluster-git/runtime/README.md

```diff
@@ -25,4 +25,6 @@ These files make up the example:
 
 * [github-for-gitops.yaml](github-for-gitops.yaml): contains configuration for connecting to a Git repository. This Resource Definition co-provisions a [GKE cluster](https://developer.humanitec.com/platform-orchestrator/examples/resource-definitions/#k8s-cluster-gke) to be used to fetch Runtime Information, the `k8s-cluster-runtime` Id is used in the co-provision key. This format is for use with the [Humanitec CLI](<https://developer.humanitec.com/>).
 * [gke-temporary-credentials-runtime.yaml](gke-temporary-credentials-runtime.yaml): uses temporary credentials defined via a Cloud Account. The Resource Id specified in the Matching Criteria is `k8s-cluster-runtime` and it ensures that this Definition will be matched to provision the `k8s-cluster` Resource co-provisioned by the GitOps cluster Resource Definition. This format is for use with the [Humanitec CLI](https://developer.humanitec.com/platform-orchestrator/cli/)
+  * This `runtime` Resource Definition can optionally use the [Humanitec Agent](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/overview) to access runtime information on a private cluster. This requires the Agent to be configured for being able to access the cluster and the corresponding Agent Resource Definition (next item) being matched in the `agent_url` property. See the [documentation](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/how-tos/deploy-to-a-private-cluster-using-the-humanitec-agent/) for details.
+* [`gke-agent.yaml`](./gke-agent.yaml): defines the Resource for the [Humanitec Agent](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/overview). Relevant only if the Agent is being used to access the runtime cluster
```

Comment:
Nit: we miss a `.` at the end of this sentence.

Reply:
Good catch, thanks! I changed it, now ready to merge.